com.unity.multiplayer.mlapi/Runtime/com.unity.multiplayer.mlapi.runtime.asmdef

```diff
@@ -1,14 +1,24 @@
 {
     "name": "Unity.Multiplayer.MLAPI.Runtime",
     "rootNamespace": "MLAPI",
-    "references": [],
+    "references": [
+        "GUID:821943bbff0df3b48b43584136c29e17",
```

Comment:
What are those references? Do we need them? Maybe use text instead of GUIDs in the asmdef?